WTG.Analyzers.Utils/WTG.Analyzers.Utils.csproj

```diff
@@ -1,12 +1,10 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AssemblyTitle>WTG Analyzers Utils</AssemblyTitle>
     <TargetFramework>netstandard1.3</TargetFramework>
     <OutputPath>..\Bin\</OutputPath>
     <PackageTargetFallback>portable-net45+win8+wp8+wpa81</PackageTargetFallback>
   </PropertyGroup>
 
-  <Import Project="..\build\Global.props" />
-  <Import Project="..\build\Global.targets" />
   <Import Project="..\.paket\Paket.Restore.targets" />
```

Comment:
Can we include Paket from Directory.Build.targets, while you're doing this?

Reply:
Yep.